SimpleWAWS/Models/Subscription.cs

```diff
@@ -2,13 +2,30 @@
 using System.Collections.Generic;
 using System.Linq;
 using System.Web;
+using SimpleWAWS.Code;
 
 namespace SimpleWAWS.Models
 {
     public class Subscription
     {
         public string SubscriptionId { get; private set; }
+        public enum SubscriptionType
```

Comment:
I wanted to comment on this throughout the PR :). It would be much better if you moved this enum out of the subscription class and make it top level.